tests/zim/test_zim_creator.py

```diff
@@ -540,6 +542,82 @@ def test_check_metadata(tmp_path):
         Creator(tmp_path, "").config_dev_metadata(LongDescription="T" * 5000).start()
 
 
+@pytest.mark.parametrize(
+    "tags",
+    [
+        (
+            "wikipedia;_category:wikipedia;_pictures:no;_videos:no;_details:yes;"
+            "_ftindex:yes"
+        ),
+        (
+            [
+                "wikipedia",
+                "_category:wikipedia",
+                "_pictures:no",
+                "_videos:no",
+                "_details:yes",
+                "_ftindex:yes",
+            ]
+        ),
+    ],
+)
+@patch("zimscraperlib.zim.creator.logger", autospec=True)
+def test_start_logs_metadata_log_contents(mocked_logger, png_image, tags, tmp_path):
+    mocked_logger.isEnabledFor.side_effect = lambda level: level == logging.DEBUG
+    fpath = tmp_path / "test_config.zim"
+    with open(png_image, "rb") as fh:
+        png_data = fh.read()
+    # Using `with creator:` would call start() twice, complicating
+    # the assert below.
+    creator = Creator(fpath, "").config_metadata(
+        Name="wikipedia_fr_football",
+        Title="English Wikipedia",
+        Creator="English speaking Wikipedia contributors",
+        Publisher="Wikipedia user Foobar",
+        Date="2009-11-21",
+        Description="All articles (without images) from the english Wikipedia",
+        LongDescription="This ZIM file contains all articles (without images)"
+        " from the english Wikipedia by 2009-11-10. The topics are...",
```

Comment:
I find this test value a bit awkward with the `...` at the end. This is not that much a nice looking long description. Please change it to something more "recommendable" (you can just end it at the end of previous sentence).

Reply:
It's a copy of the metadata from the nearby test_config_metadata. Do you really want this changed, and do you want me to change that one while I'm at it?

IIUC, LongDescriptor is intended to be longer than the other descriptors, so the formatting awkwardness is expected. And in fact this is a lot shorter than the actual long description.

Reply:
Oh, right, let's live with it then.